bigbluebutton-client/src/org/bigbluebutton/main/model/users/Conference.as

```diff
@@ -197,8 +198,16 @@ package org.bigbluebutton.main.model.users {
 				}
 			}
 			return null;
-		}
-		
+        }
+
+        public function userIsModerator(userId:String):Boolean {
+            var moderator:BBBUser = getTheOnlyModerator();

```

Comment:
Using `getUser(userId)` is better. I will fix that. However we need to keep the check for `user != null`.
I will update it to `return user != null && user.role == Role.MODERATOR;`